### DISCUSSION TITLE
Latest Python

### DISCUSSION THREAD

**Discussion by hassamohammed** [community/community#145987] [Programming Help]
### Select Topic Area

Question

### Body

Does anyone know the latest python version usuable with most PIP packages?

**Comment by hassamohammed** [comment]
So far all I know works is 3.7 because I am scared to try anything newer. Google said it reccommends to try 3.10, so I will do so, nevermind, thanks.

**Reply by NickOsipov** [reply]
@hassamohammed 
Hello! I'm used 3.11.4 or 3.11.7 and it's okay with all my dependencies needs.

**Reply by mecodeatlas** [reply]
Hey, @hassamohammed! Did @NickOsipov's response help you resolve your question?
